To: vim_dev@googlegroups.com
Subject: Patch 7.3.238
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.238
Problem:    Compiler warning for conversion.
Solution:   Add type cast. (Mike Williams)
Files:	    src/ex_getln.c


*** ../vim-7.3.237/src/ex_getln.c	2011-06-26 19:40:14.000000000 +0200
--- src/ex_getln.c	2011-07-04 14:23:27.000000000 +0200
***************
*** 5025,5031 ****
  
      *num_file = 0;
      *file = NULL;
!     pat_len = STRLEN(pat);
      ga_init2(&ga, (int)sizeof(char *), 10);
  
      for (i = 0; dirnames[i] != NULL; ++i)
--- 5025,5031 ----
  
      *num_file = 0;
      *file = NULL;
!     pat_len = (int)STRLEN(pat);
      ga_init2(&ga, (int)sizeof(char *), 10);
  
      for (i = 0; dirnames[i] != NULL; ++i)
*** ../vim-7.3.237/src/version.c	2011-06-26 19:40:14.000000000 +0200
--- src/version.c	2011-07-07 15:04:00.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     238,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
255. You work for a newspaper and your editor asks you to write an
     article about Internet addiction...in the "first person."

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
